cpp/include/raft/core/detail/macros.hpp

```diff
@@ -86,33 +86,7 @@
 // as a weak symbol rather than a global."
 #define RAFT_WEAK_FUNCTION __attribute__((weak))
 
-// The RAFT_HIDDEN_FUNCTION specificies that the function will be hidden
```

Comment:
This can't be removed without changes to cuvs first.

Since cuvs uses the `RAFT_KERNEL` macro and builds some components with RDC enabled we need to keep this logic. CUVS explicitly decided to need re-implement the macro logic and instead leverage the RAFT versions

Reply:
Thanks for pointing this out! I re-added the macros.